apps/dashboard/src/components/onboarding/ChoosePlan.tsx

```diff
@@ -1,118 +1,61 @@
-        onSuccess: () => {
-          onSave();
-
-          trackEvent({
-            category: "account",
-            action: "choosePlan",
-            label: "success",
-            data: {
-              plan,
-            },
-          });
-        },
-        onError: (error) => {
-          trackEvent({
-            category: "account",
-            action: "choosePlan",
-            label: "error",
-            error,
-          });
-        },
-      },
-    );
-  };
-
+function OnboardingChoosePlan(props: {
+  skipPlan: () => void;
+}) {
   return (
-    <>
+    <div>
       <TitleAndDescription
         heading="Choose your plan"
         description="Get started for free with our Starter plan or subscribe to Growth plan to unlock higher rate limits and advanced features."
       />
-      <div className="grid grid-cols-1 gap-8 md:grid-cols-2">
+
+      <Button
+        variant="ghost"
+        onClick={props.skipPlan}
+        className="absolute top-4 right-4 inline-flex items-center gap-2"
+      >
+        Continue with Free Plan <ArrowRightIcon className="size-4" />
+      </Button>
+
+      <div className="h-4" />
+
+      <div className="grid grid-cols-1 gap-6 md:grid-cols-2">
         <PricingCard
-          size="sm"
-          name={accountPlan.free}
-          ctaTitle="Get started for free"
-          ctaProps={{
-            category: "account",
-            onClick: (e) => {
-              e.preventDefault();
-              handleSave(accountPlan.free);
+          billingPlan="starter"
+          cta={{
+            title: "Get started for free",
+            // directly redirect to stripe on onboarding
+            href: "/team/~/subscribe/plan:starter",
+            tracking: {
+              category: "account",
             },
-            label: "freePlan",
-            href: "/",
           }}
-          onDashboard
         />
 
         <PricingCard
-          size="sm"
-          ctaTitle="Claim your 1-month free"
-          name={accountPlan.growth}
-          ctaHint="Your free trial will end after 30 days."
-          canTrialGrowth={true}
-          ctaProps={{
-            category: "account",
-            label: "growthPlan",
-            onClick: (e) => {
-              e.preventDefault();
-              handleSave(accountPlan.growth);
+          billingPlan="growth"
+          cta={{
+            title: "Claim your 1-month free",
+            hint: "Your free trial will end after 30 days.",
+            tracking: {
+              category: "account",
+              label: "growthPlan",
             },
-            href: "/",
-            variant: "solid",
-            colorScheme: "blue",
+            // directly redirect to stripe on onboarding
+            href: "/team/~/subscribe/plan:growth",
+            variant: "default",
           }}
-          onDashboard
+          // Question: @Jonas
+          // Is this true? - all new users can try grow for 1 month?
+          canTrialGrowth={true}
```

Comment:
@jnsdls question for you here

Reply:
if they have never had growth plan before then yes - I'll update this to be included in the api response so we can drive it from that

Reply:
I see the usage of `!account.trialPeriodEndedAt` for determining  this - does that work?